account_invoice_production_lot/models/account_invoice.py

```diff
@@ -51,6 +62,7 @@ def _compute_line_lots(self):
                 note += u'</ul>'
                 line.lot_formatted_note = note
 
+    @api.multi
```

Comment:
There's no need to put the decorator: it's the default one when there's nothing.

Reply:
ok, good to know it, but according to the Zen of Python `Explicit is better than implicit.` :wink:

Reply:
But this is not Python, but Odoo ORM :wink: